lib/chef/util/dsc/local_configuration_manager.rb

```diff
@@ -72,10 +72,24 @@ def run_configuration_cmdlet(configuration_document, apply_configuration, shello
       status
     end
 
-    def lcm_command_code(configuration_path, test_only_parameters)
-      <<-EOH
-$ProgressPreference = 'SilentlyContinue';start-dscconfiguration -path #{@configuration_path} -wait -erroraction 'stop' -force #{test_only_parameters}
+    def lcm_command_code(configuration_path, test_command)
+      if test_command == "test"
+        <<-EOH
+$ProgressPreference = 'SilentlyContinue';Test-DscConfiguration -path #{@configuration_path}
 EOH
+      else
+        <<-EOH
+$ProgressPreference = 'SilentlyContinue';start-dscconfiguration -path #{@configuration_path} -wait -erroraction 'stop' -force #{test_command}
+EOH
+      end
+    end
+
+    def test_lcm_command_code
+      if Chef::Platform.supported_powershell_version?(@node, 5)
+        "test"
```

Comment:
Can we return something more specific to Powershell version 5 e.g. `test_powershell_5`? Returning `test` looks vague here.